cpp/README.md

```diff
@@ -0,0 +1,80 @@
+# Sift C++
+
+This tutorial shows how to build a C++ application that uses the Sift gRPC API.
+
+## Installation
+
+##### Build gRPC and Protocol Buffers
+
+Follow the [official gRPC C++ tutorial](https://grpc.io/docs/languages/cpp/quickstart/#install-grpc) to build and locally install gRPC and Protocol Buffers. You should be able to build and run the [helloworld](https://grpc.io/docs/languages/cpp/quickstart/#build-the-example) example before continuing with the next section.
+
+Ensure that `$ which grpc_cpp_plugin` and `$ which protoc` generate a path to executables.
+
+You will also need to ensure that you have the [buf CLI](https://buf.build/docs/installation) installed.
+
+Verify that `$ which buf` generates a path to the executable, before proceeding to the compilation steps.
+
+##### C++ Protobuf Compilation
```

Comment:
super nitty nit, think it'd be nice to cross-link them to the docs for our protos in one of these sections? maybe as a like: "now that you have the protos compiled, refer to docs to find more info" sort of thing.